glean-core/ios/Glean/Metrics/TimespanMetric.swift

```diff
@@ -56,7 +56,7 @@ public class TimespanMetricType {
 
         let startTime = timestampNanos()
 
-        _ = Dispatchers.shared.launch {
+        _ = Dispatchers.shared.launchAPI {
```

Comment:
As above and below a couple of times. no `_ =` needed

Reply:
There were several places this still existed, I squashed them all.